SpatialGDK/Source/SpatialGDK/Private/SpatialView/OpList/EntityComponentOpList.cpp

```diff
@@ -195,4 +209,9 @@ OpList EntityComponentOpListBuilder::CreateOpList() &&
 {
 	return { OpListData->Ops.GetData(), static_cast<uint32>(OpListData->Ops.Num()), MoveTemp(OpListData) };
 }
+
+TArray<Worker_Op> EntityComponentOpListBuilder::CreateOpArray() &&
```

Comment:
I don't think it makes sense to have this as a rvalue only function. That would mean all the data storage might be deleted immediately. Why not just use CreateOpList?

Reply:
the Handler takes as input an array of ops instead of an oplist. I mainly wanted to avoid having to convert from OpList to a TArray<Op>